test/foo.test.js

```diff
@@ -1,13 +1,12 @@
 /*global describe, it*/
-'use strict';
 
-var expect = require('expect.js');
+const expect = require('expect.js');
 
-var Foo = require('./../src/js/foo');
+const foo = require('./../src/js/foo');
```

Comment:
Capitals should be reserved for constructors. In this example it is just a function? It's not doing anything constructor-like.